horizomer/utils/tree.py

```diff
@@ -15,6 +15,77 @@
 from skbio import TreeNode
 
 
+def order_nodes(tree, increase=True):
+    """Restore ordering of nodes in one tree based on another tree.
+
+    Parameters
+    ----------
+    tree : skbio.TreeNode
+        tree to order
+    increase : bool, optional
+        order nodes in increasing (True) or decreasing (False) order
+
+    Returns
+    -------
+    skbio.TreeNode
+        resulting ordered tree
+    """
+    res = tree.copy()
+    for node in res.postorder():
```

Comment:
I am thinking about moving this loop out into an independent function. But this is not urgent. I can take care of this.